src/transformers/models/blip_2/configuration_blip_2.py

```diff
@@ -91,7 +91,7 @@ def __init__(
         image_size=224,
         patch_size=14,
         hidden_act="gelu",
-        layer_norm_eps=0.00001,
+        layer_norm_eps=1e-6,
```

Comment:
Does this mean that the values for existing config files on the hub need to be updated?

Reply:
We could do that, although it would be a minor breaking change.

Reply:
The question is whether the configs should have had this value in the first place. If so, then the official checkpoints should have their values updated.